_benchmark/user-guide/understanding-workloads/choosing-a-workload.md

```diff
@@ -20,10 +20,76 @@ Consider the following criteria when deciding which workload would work best for
+For benchmarking clusters built for general search use cases, start with the [nyc_taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/nyc_taxis) workload. It contains the contains the following:
 
-## Log data
+- **Data type:** Ride data from yellow taxis in New York City in 2015.
+- **Cluster requirements:** Suitable for small to medium-sized clusters.
 
-For benchmarking clusters built for indexing and search with log data, use the [http_logs](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/http_logs) workload. This workload contains data about the 1998 World Cup.
+This workload tests the following queries and search functions:
+
+- Range queries.
+- Term queries on various fields.
+- Geo-distance queries.
+- Aggregations.
+
+## Vector data: `vectorsearch`
+
+The [`vectorsearch`](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/vectorsearch)  workload is designed to benchmark vector search capabilities, including performance and accuracy. It contains the following:
+
+- **Data type:** High-dimensional vector data, often representing embeddings of text or images.
+- **Cluster requirements:** Requires a cluster with vector search capabilities enabled.
+
+This workload tests the following queries and search functions:
+
+- k-NN vector searches.
+- Hybrid searches combining vector similarity with metadata filtering.
+- Indexing performance for high-dimensional vector data.
+
+## Comprehensive search solutions: `big5`
+
+The [big5](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/big5) workload is a comprehensive benchmark suite for testing various aspects of search engine performance, including overall search engine performance across multiple use cases. It contains the following:
+
+- **Data type:** A mix of different data types, including text, numeric, and structured data.
+- **Cluster requirements:** Suitable for medium to large clusters, as it's designed to stress-test various components.
+
+This workload tests the following queries and search functions:
+
+- Full-text search performance.
+- Aggregation performance.
+- Complex Boolean queries.
+- Sorting and pagination.
+- Indexing performance for various data types.
+
+## Percolator queries: `percolator`
+
+The [percolator](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/percolator) workload is designed to test the performance of the percolator query type. It contains the following:
+
+- **Data type:** A set of stored queries and documents to be matched against those queries.
+- **Cluster requirements:** Suitable for clusters that make heavy use of the percolator feature.
```

Comment:
Might be nice to link percolator documentation in OpenSearch

Reply:
@IanHoang: The field type documentation here, https://opensearch.org/docs/latest/field-types/supported-field-types/percolator/?